src/test/java/org/folio/circulation/domain/InstanceRequestItemsComparerTests.java

```diff
@@ -0,0 +1,258 @@
+package org.folio.circulation.domain;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTime;
+import org.junit.Test;
+
+import java.util.*;

```

Comment:
Maybe use explicit imports here instead of the import glob.

Reply:
Changed.